src/xcode/ENA/ENA/Source/Services/HealthCertificateValidation/__tests__/HealthCertificateValidationServiceValidationTests.swift

```diff
@@ -1527,3 +1635,22 @@ extension RuleValidationError: Equatable {
 		}
 	}
 }
+
+private class ValueSetsStub: VaccinationValueSetsProviding {
+
+	var valueSets: SAP_Internal_Dgc_ValueSets!
```

Comment:
If you made this a `struct` you could get the `init` for free, would be a bit nicer instead of the implicitly unwrapped I think.